mw-webapp/src/dataAccessLogic/useGetDataTableReports.ts

```diff
@@ -1,18 +1,31 @@
 import {useEffect, useState} from "react";
 import {DayReportDAL} from "src/dataAccessLogic/DayReportDAL";
 import {DayReport} from "src/model/businessModel/DayReport";
+import {WayService} from "src/service/WayService";
 
 /**
- * Load day reports
+ * Load day reports of specific Way if wayUuid is provided, otherwise load all day reports
  */
-export const useGetDataTableReports = () => {
+export const useGetDataTableReports = (wayUuid?: string) => {
```

Comment:
it will look very strange - to render all DayReports if some errors with uuid. DayReport collection will include millions day reports of different ways. I think that uuid prop shouldn't be optional.

Reply:
Did not undrestand what you mean with "render all DayReports if some errors with uuid". I just refactored this function to be able to render specific Way. If you don't like it, what should be rendered by default if user is not authorized?

Reply:
If user is not authorized then first it should be AuthPage and then allUsers page or allWays page

Reply:
So i should redirect user from main page if he is not authorized?